src/schema/v2/ArtworkImport/artworkImport.ts

```diff
@@ -163,6 +169,12 @@ export const ArtworkImportType = new GraphQLObjectType({
         },
       }),
       resolve: async ({ id, currency }, args, { artworkImportRowsLoader }) => {
+        if (!artworkImportRowsLoader) {
```

Comment:
Not sure how to get around this @mzikherman. Would love to hear your thoughts! Had to use thunk to get the date and this then caused the loader to potentially be `undefined` causing this code. I don't think this should ever happen because this is already authorized territory but couldn't fix this in another way.

Reply:
This is fine!